docs/install/manual-install.md

```diff
@@ -132,32 +131,12 @@ and compiles any translation `.mo` files (using `commonlib/bin/gettext-makemo`).
 
 #### macOS troubleshooting
 
-Some Perl modules may fail to install without certain packages and paths in place.
-
-First make sure you have installed any [prerequisite packages](#pre-req-pkg-mac).
-
-For any **SSL-related modules**, you may need to create symlinks to folders Perl modules will recognise:
-
-{% highlight bash %}
-$ sudo mkdir /usr/local/lib
```

Comment:
Are these commands definitely not needed? I seem to remember having to run them.

Reply:
As above, I hope that OPENSSL_PREFIX should be all that is needed, but I assume there are variations, it's hard to know directly. But I did not need to do anything to /usr/local or use sudo in order to install all the packages.

Reply:
I guess I just don’t want to encourage something (symlinking things into /usr/local) that involves something that may have unforeseen consequences (what if something else gets confused with them there), unless truly necessary. e.g. brew says: "Linking keg-only OpenSSL means you may end up linking against the insecure,
deprecated system version while using the headers from the Homebrew version." There’s lots of random things round the internet that say sudo things into places or other ‘magic’, I don’t like them much :) I have run these instructions from scratch multiple times on a fresh Mac, so I think I'd want more concrete evidence as to why these steps may be necessary, e.g. with a scenario where a Perl module did not install without this. Like I say, they all do install here with nothing in /usr/local.

Reply:
Well I can't remember what exactly was the 'fix' for me when I ran these things; it could have been one thing or a combination. If a newcomer has trouble setting things up maybe we'll have to change these docs again.